projects/app-ziti-console/src/app/app-routing.module.ts

```diff
@@ -59,12 +74,7 @@ const routes: Routes = [
     canActivate: mapToCanActivate([AuthenticationGuard]),
     runGuardsAndResolvers: 'always',
   },
-  {
-    path: 'routers',
-    component: EdgeRoutersPageComponent,
-    canActivate: mapToCanActivate([AuthenticationGuard]),
-    runGuardsAndResolvers: 'always',
-  },
+
```

Comment:
Looks like this might have been a conflict with the last merge. We don't want to remove these blocks for the new Routers list page.